src/main/scala-2.11/org/apache/james/gatling/control/JamesWebAdministration.scala

```diff
@@ -2,14 +2,15 @@ package org.apache.james.gatling.control
 
 import java.net.URL
 
+import org.apache.james.gatling.utils.RandomStringGenerator
 import play.api.libs.ws.WSResponse
 import play.api.libs.ws.ning.NingWSClient
 
 import scala.concurrent.Future
 
-case class Domain(value: String)
+case class Domain(value: String = RandomStringGenerator.randomString)
```

Comment:
generating a random value for domain (and other classes) doesn't sound like an expected default behavior.
You should have a factory with a named method like `Domain.generateRandom()` (Domain being an object in that case)